2014-05-09 Kai Pastor (OpenOrienteering)
- Fix windeployqt linking issue on non-win systems
- cf. QTBUG-38886

diff -up qttools/src/windeployqt/utils.cpp.orig qttools/src/windeployqt/utils.cpp
--- qttools/src/windeployqt/utils.cpp.orig	2014-05-09 07:23:18.752596038 +0200
+++ qttools/src/windeployqt/utils.cpp	2014-05-09 07:24:17.639859307 +0200
@@ -872,7 +872,7 @@ QString findD3dCompiler(Platform platfor
 #else // Q_OS_WIN
 
 bool readPeExecutable(const QString &, QString *errorMessage,
-                      QStringList *, unsigned *, bool *)
+                      QStringList *, unsigned *, bool *, bool)
 {
     *errorMessage = QStringLiteral("Not implemented.");
     return false;
